crates/extensions/tedge_health_ext/src/actor.rs

```diff
@@ -0,0 +1,48 @@
+use async_trait::async_trait;
+use tedge_actors::Actor;
+use tedge_actors::ReceiveMessages;
+use tedge_actors::RuntimeError;
+use tedge_actors::SimpleMessageBox;
+use tedge_mqtt_ext::MqttMessage;
+
+use tedge_api::health_status_down_message;
+use tedge_api::health_status_up_message;
```

Comment:
Note. These two `use` could be `pub`.

This is no more required with the `HealthMonitorBuilder::set_init_and_last_will()`

Reply:
done